### PR TITLE
Add links to make physical types more discoverable

### PR DESCRIPTION
**Pull request by namurphy**
### Description

The purpose of this PR is to add a few links from functions/attributes related to physical types to documentation for the `astropy.units.physical` module, which lists out the contents of that module including physical types. Since my usual strategy for finding the listing of physical types has been to do a search for "absement", this should make the listing of physical types more discoverable.

I'll probably add a few more links (i.e. in `UnitBase.physical_type` and the notebook section on physical types), and I also need to make sure that I'm linking to the correct location in the docs. 

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Comment by nstarman** [comment]
LGTM. For after v5.2, of course. Ping when you want a review.

File: astropy/units/physical.py [hunk 1]
```diff
@@ -210,6 +210,9 @@ class PhysicalType:
     `~astropy.units.core.UnitBase.physical_type` attribute of units.
     This class is not intended to be instantiated directly in user code.
 
+    To learn what physical types are available, please refer to the
```
Comment (mhvk) [inline astropy/units/physical.py:213]: I'd keep this shorter perhaps "For a list of physical types, see `astropy.units.physical`." (and same in other instances)

**Review by mhvk — APPROVED** [review]
Looks good as is, so approving, though perhaps it can be a bit more concise.

File: astropy/units/physical.py (hunk 1 above)
Comment (namurphy) [reply astropy/units/physical.py:213]: Good idea!  Let's go with your wording.

**Comment by namurphy** [comment]
I just rebased after making the sentences more concise. This should be now ready to merge.

**Comment by mhvk** [comment]
OK, thanks, let's get it in!

**Pull request merged**